{"name": "Проверка ошибки авторизации курьера: авторизация несуществующего курьера", "status": "passed", "start": 1699616040939, "stop": 1699616041147, "uuid": "327d07d9-6253-4fcd-af3d-d1bb5e03226f", "historyId": "65308b128bf99f45f905d42de5bba36f", "testCaseId": "65308b128bf99f45f905d42de5bba36f", "fullName": "tests.test_courier_login.TestCourierLogin#test_courier_login_nonexistent_courier_failed_login", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "WKS-084"}, {"name": "thread", "value": "20968-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}]}